contributing.md

```diff
@@ -45,14 +45,25 @@ an explicit [Code of Conduct](https://github.com/ampproject/amp-wp/blob/develop/
 To start, clone this repository into your WordPress install being used for development:
 
 ```bash
-cd wp-content/plugins && git clone --recursive git@github.com:ampproject/amp-wp.git amp
+cd wp-content/plugins && git clone git@github.com:ampproject/amp-wp.git amp
 ```
 
-If you happened to have cloned without `--recursive` previously, please do `git submodule update --init` to ensure the [dev-lib](https://github.com/xwp/wp-dev-lib/) submodule is available for development.
+Then install the packages:
+
+```bash
+composer install
+npm install
+```
+
+And lastly, do a build of the JavaScript:
+
+```bash
+npm run build
+```
 
 Lastly, to get the plugin running in your WordPress install, run `composer install` and then activate the plugin via the WordPress dashboard or `wp plugin activate amp`.
 
-To install the `pre-commit` hook, do `bash dev-lib/install-pre-commit-hook.sh`.
+To install the `pre-commit` hook, do `bash vendor/xwp/wp-dev-lib/scripts/install-pre-commit-hook.sh`.
```

Comment:
@westonruter Maybe use `brainmaestro/composer-git-hooks` with Composer or husky with npm [per dev-lib docs](https://github.com/xwp/wp-dev-lib/tree/a458f7915c672e0d6431499040d6d8cd135a532a#configure-the-git-pre-commit-hook)?

Reply:
> Maybe use `brainmaestro/composer-git-hooks` with Composer or `husky` with npm per dev-lib docs?

+1

Reply:
Agreed. However, given that the `pre-commit` hook doesn't work yet with Lando, I was thinking to defer this to another PR.